.drone.yml

```diff
@@ -281,6 +281,37 @@ trigger:
     - pull_request
 
 ---
+# When our UI and Unit tests complete, we trigger a GitHub Actions workflow
+# to do followup work.
+kind: pipeline
+type: docker
+name: on-droneci-complete
+clone:
+  disable: true
+steps:
+  - name: trigger-github-actions
+    image: plugins/webhook
+    settings:
+      urls:
+        - https://api.github.com/repos/code-dot-org/code-dot-org/actions/workflows/on-droneci-complete.yml/dispatches
+      username: deploy-code-org
+      password:
+        from_secret: on-droneci-complete-github-token
+      template: |
+        {
+          "ref": "${DRONE_SOURCE_BRANCH}",
+          "inputs": {
+            "github_sha": "{{ build.commit }}",
+            "drone_build_status": "{{ build.status }}"
+          }
+        }
+# depends_on:
+#   - unit
+#   - ui
```

Comment:
Note to self: make sure we uncomment / re-enable `depends_on: ['unit', 'ui']` before merging.